components/list/demo/loadmore.ts

```diff
@@ -18,19 +18,21 @@ const fakeDataUrl = 'https://randomuser.me/api/?results=5&inc=name,gender,email,
           @if (item.loading) {
             <nz-skeleton [nzAvatar]="true" [nzActive]="true" [nzTitle]="false" [nzLoading]="true" />
           } @else {
-            <nz-list-item-meta
-              nzAvatar="https://zos.alipayobjects.com/rmsportal/ODTLcjxAfvqbxHnVXCYX.png"
-              nzDescription="Ant Design, a design language for background applications, is refined by Ant UED Team"
-            >
-              <nz-list-item-meta-title>
-                <a href="https://ng.ant.design">{{ item.name.last }}</a>
-              </nz-list-item-meta-title>
-            </nz-list-item-meta>
-            content
-            <ul nz-list-item-actions>
-              <nz-list-item-action><a (click)="edit(item)">edit</a></nz-list-item-action>
-              <nz-list-item-action><a (click)="edit(item)">more</a></nz-list-item-action>
-            </ul>
+            <ng-container>
+              <nz-list-item-meta
+                nzAvatar="https://zos.alipayobjects.com/rmsportal/ODTLcjxAfvqbxHnVXCYX.png"
+                nzDescription="Ant Design, a design language for background applications, is refined by Ant UED Team"
+              >
+                <nz-list-item-meta-title>
+                  <a href="https://ng.ant.design">{{ item.name.last }}</a>
+                </nz-list-item-meta-title>
+              </nz-list-item-meta>
+              content
+              <ul nz-list-item-actions>
+                <nz-list-item-action><a (click)="edit(item)">edit</a></nz-list-item-action>
+                <nz-list-item-action><a (click)="edit(item)">more</a></nz-list-item-action>
+              </ul>
+            </ng-container>
```

Comment:
Is this an accidental change? It doesn't look like <ng-container> is needed here.

Reply:
angular output warning like this

```
▲ [WARNING] NG8011: Node matches the "nz-list-item-meta, [nz-list-item-meta]" slot of the "NzListItemComponent" component, but will not be projected into the specific slot because the surrounding @else has more than one node at its root. To project the node in the right slot, you can:

1. Wrap the content of the @else block in an <ng-container/> that matches the "nz-list-item-meta, [nz-list-item-meta]" selector.
2. Split the content of the @else block across multiple @else blocks such that each one only has a single projectable node at its root.
3. Remove all content from the @else block, except for the node being projected.

This check can be disabled using the `extendedDiagnostics.checks.controlFlowPreventingContentProjection = "suppress" compiler option.` [plugin angular-compiler]

    site/doc/app/list/loadmore.ts:21:12:
      21 │             <nz-list-item-meta
         ╵             ~~~~~~~~~~~~~~~~~~

```